src/LibVLCSharp.WPF/VideoView.cs

```diff
@@ -4,21 +4,17 @@
 using System.Diagnostics;
 using System.Windows;
 using System.Windows.Controls;
-using System.Windows.Forms.Integration;
 
 namespace LibVLCSharp.WPF
 {
     /// <summary>
     /// WPF VideoView with databinding for use with LibVLCSharp
     /// </summary>
-    [TemplatePart(Name = PART_PlayerHost, Type = typeof(WindowsFormsHost))]
-    [TemplatePart(Name = PART_PlayerView, Type = typeof(System.Windows.Forms.Panel))]
+    [TemplatePart(Name = PART_PlayerHost, Type = typeof(ControlHwndHost))]
```

Comment:
What is preventing us from putting HwndHost here and at `if (Template.FindName(PART_PlayerHost, this) is not ControlHwndHost controlHost)` ?
Do we need something specific from that class ? Can anyone use its own implementation instead of the default one by overriding the template ?

Reply:
I didn't get this, if you mean why did we use ControlHwndHost inside the template? I should tell you, I have followed the previous code, but if you mean why did I remove the Template.FindName(PART_PlayerHost, this) from the public property, because It will call FindName function which I think it doesn't really need it and should call on ApplyTemplate!

Reply:
I meant replace this line with:
```cs
[TemplatePart(Name = PART_PlayerHost, Type = typeof(HwndHost))]
```

and replace the line:
```cs
if (Template.FindName(PART_PlayerHost, this) is not ControlHwndHost controlHost)
```

with 
```cs
if (Template.FindName(PART_PlayerHost, this) is not HwndHost windowHost)
```